src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let countCalls = 0;
+
+  return (firstNumber) => {
+    return (secondNumber) => {
+      countCalls++;
+
+      if (countCalls <= 3 || countCalls % 2 !== 0) {
```

Comment:
The condition should be countCalls > 3 and countCalls % 2 === 0 for even calls to return an error.